examples/grid_packing/main.rs

```diff
@@ -0,0 +1,41 @@
+use gtk::glib::clone;
+use gtk::prelude::*;
+
+fn build_ui(application: &gtk::Application) {
+    // Create a new window, and set its title
+    let window = gtk::ApplicationWindow::new(application);
```

Comment:
I would use ApplicationWindowBuilder & set a larger window default size, see other examples

Reply:
Like in the original C example, the buttons and the grid don't expand with the size of the window. Should I add that too if we're going for a window that's bigger than the widgets inside?

Reply:
Yeah, there's no need to expand the grid :) it would just look ugly

Reply:
So you want me to have a larger window, but not have the widgets inside expand? It'll probably look something like this:
![Screenshot from 2021-04-30 14-44-21](https://user-images.githubusercontent.com/45475651/116752225-795fc080-a9c2-11eb-83cb-5f87a30bd554.png)

Are you sure we want this?

Reply:
Set the valign/halign on the grid to center, so it's centered on the window

Reply:
Okay I'll do that

Reply:
And probably set column/row spacing so that the items don't look so close to each other. That would demo another aspect of GtkGrid I guess

Reply:
Make sure to update the screenshot afterwards

Reply:
Column and row spacing as well as an updated screenshot should be there now